tests/manual-test-cases/Group5-Functional-Tests/5-4-High-Availability.robot

```diff
@@ -17,6 +17,78 @@ Documentation  Test 5-4 - High Availability
 Resource  ../../resources/Util.robot
 Suite Teardown  Nimbus Cleanup  ${list}
 
+*** Keywords ***
+Check VM Info
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.info \*
+    Should Be Equal  ${rc}  0
+    Log To Console  ${output}
+
+Check ImageStore
+    ${rc}  ${output}=  Run And Return Rc And Output  govc datastore.ls -R -ds=nfsDatastore %{VCH-NAME}/VIC
+    Should Be Equal  ${rc}  0
+    Log To Console  ${output}
```

Comment:
log not log to console
-ds should be %{TEST_DATASTORE}
also, if it doesn't exist already this seems valuable to be placed in resources/vsphere-util